prime-router/src/main/kotlin/transport/FTPSTransport.kt

```diff
@@ -187,21 +188,18 @@ class FTPSTransport : ITransport, Logging {
             fileName: String,
             contents: ByteArray
         ): Boolean {
+            val logger = logger("gov.cdc.prime.router.transport.FTPSTransport")
             try {
                 val fileInputStream: InputStream = ByteArrayInputStream(contents)
                 return ftpsClient.storeFile(fileName, fileInputStream)
             } catch (e: FTPConnectionClosedException) {
-                println("Server closed connection.")
-                e.printStackTrace()
+                logger.warn("Server closed connection\n" + e.stackTraceToString())
             } catch (e: CopyStreamException) {
-                println("Error while transferring file to FTPS server")
-                e.printStackTrace()
+                logger.warn("Error while transferring file to FTPS server\n" + e.stackTraceToString())
             } catch (e: IOException) {
-                println("IOException")
-                e.printStackTrace()
+                logger.warn("IOException (" + e.toString() + ")\n" + e.stackTraceToString())
             } catch (e: Exception) {
-                println("General exception")
-                e.printStackTrace()
+                logger.warn("General exception (" + e.toString() + ")\n" + e.stackTraceToString())
```

Comment:
note that I made these `warn` instead of `error` based on conversation yesterday in Pipeline grooming around potentially sending error/fatal to PagerDuty